tasks/unit-tests/modules_test.py

```diff
@@ -0,0 +1,81 @@
+import json
+import os
+import subprocess
+import unittest
+from typing import Any, Set
+
+from ..modules import DEFAULT_MODULES
+
+AGENT_MODULE_PATH_PREFIX = "github.com/DataDog/datadog-agent/"
+
+
+class TestModules(unittest.TestCase):
+    def load_go_mod(self, module_path: str) -> Any:
+        """Loads the go.mod file as a JSON object"""
+        go_mod_path = os.path.join(module_path, "go.mod")
+        res = subprocess.run(["go", "mod", "edit", "-json", go_mod_path], capture_output=True)
```

Comment:
Maybe adding a comment showing a JSON example of calling `go mod edit -json <mod>` would help when looking at the test logic

Reply:
Sure